Xamarin.Forms.Core/Element.cs

```diff
@@ -359,6 +358,9 @@ protected override void OnBindingContextChanged()
 
 		protected virtual void OnChildAdded(Element child)
 		{
+			Dispatcher = DispatcherManager.Current.Dispatcher;
```

Comment:
@wachs - Isn't Dispatcher already set in the constructor? Is there any reason it needs to be updated here?

Reply:
Also ends up not being necessary.